packages/react/src/CounterLabel/CounterLabel.tsx

```diff
@@ -14,42 +19,74 @@ export type CounterLabelProps = React.PropsWithChildren<
 >
 
 const CounterLabel = forwardRef<HTMLSpanElement, CounterLabelProps>(
-  ({scheme = 'secondary', sx = defaultSxProp, children, className, ...props}, forwardedRef) => {
+  ({scheme = 'secondary', sx = defaultSxProp, className, children, ...rest}, forwardedRef) => {
+    const enabled = useFeatureFlag('primer_react_css_modules_team')
+    const label = <VisuallyHidden>&nbsp;({children})</VisuallyHidden>
+    const counterProps = {
+      ref: forwardedRef,
+      ['aria-hidden']: 'true' as const,
+      ['data-scheme']: scheme,
+      ...rest,
+    }
+
+    if (enabled) {
+      if (sx !== defaultSxProp) {
+        return (
+          <>
+            <Box as="span" {...counterProps} className={clsx(className, classes.CounterLabel)} sx={sx}>
+              {children}
+            </Box>
+            {label}
+          </>
+        )
+      }
+      return (
+        <>
+          <span {...counterProps} className={clsx(className, classes.CounterLabel)}>
+            {children}
+          </span>
+          {label}
+        </>
+      )
+    }
+
     return (
       <>
-        <Box
-          aria-hidden="true"
-          className={className}
-          sx={merge<BetterSystemStyleObject>(
-            {
-              display: 'inline-block',
-              padding: '2px 5px',
-              fontSize: 0,
-              fontWeight: 'bold',
-              lineHeight: 'condensedUltra',
-              borderRadius: '20px',
-              backgroundColor: scheme === 'primary' ? 'neutral.emphasis' : 'neutral.muted',
-              border:
-                'var(--borderWidth-thin,max(1px, 0.0625rem)) solid var(--counter-borderColor,var(--color-counter-border))',
-              color: scheme === 'primary' ? 'fg.onEmphasis' : 'fg.default',
-              '&:empty': {
-                display: 'none',
-              },
-            },
-            sx,
-          )}
-          {...props}
-          as="span"
-          ref={forwardedRef}
-        >
+        <StyledCounterLabel {...counterProps} className={className} sx={sx}>
           {children}
-        </Box>
-        <VisuallyHidden>&nbsp;({children})</VisuallyHidden>
+        </StyledCounterLabel>
+        {label}
       </>
     )
   },
 )
 
+const StyledCounterLabel = styled.span`
+  display: inline-block;
+  padding: var(--base-size-2, 0.125rem) var(--base-size-6, 0.375rem);
+  font-size: 12px;
+  font-weight: bold;
```

Comment:
```suggestion
  font-weight: var(--base-text-weight-semibold, bold);
```

I wonder if this might be failing tests but just guessing